### ISSUE TITLE
Survey task choices: visual layout doesn't reflect source order

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/Panoptes-Front-End#3564]
## Expected behavior
Tabbing through survey task choices should move sequentially down each column in turn. eg. for Chicago Wildlife Watch, the order should be Beaver, Bird, Cat (domestic), Chipmunk etc.

## Current behavior
The actual tab order of the choices on CWW is Beaver,  Human, Skunk, Bird ie. the choices are actually laid out in rows across the page, even though we read them vertically.

## Steps to replicate
Select choices in a survey task project using the tab key.

**Comment by eatyourgreens** [comment]
#3623 partly addresses this by using the up/down cursor keys to navigate the list in alphabetical order.
